src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -160,6 +249,22 @@ void GetInteriorVehicleDataRequest::on_event(
 
 GetInteriorVehicleDataRequest::~GetInteriorVehicleDataRequest() {}
 
+std::vector<application_manager::ApplicationSharedPtr>
+GetInteriorVehicleDataRequest::AppsSubscribedToModuleType(
```

Comment:
we have this `AppsSubscribedToModuleType` in the RcHelper. 
please re-use that one.

Reply:
@yang1070 fixed bfaf71f

Reply:
@LuxoftAKutsan AppsSubscribedToModuleType still here

Reply:
@yang1070 Fixed 
2e1171b